packages/ui-tests/cypress/e2e/designer/catalogs/userSpecifiedCatalog.cy.ts

```diff
@@ -0,0 +1,49 @@
+describe('Tests for user specified Quarkus catalog type', () => {
+  // Specify custom catalog URL
+  before(() => {
+    cy.openHomePage();
+    cy.openSettings();
+    cy.interactWithConfigInputObject(
+      'catalogUrl',
+      'https://raw.githubusercontent.com/tplevko/kaoto-catalog-test/main/index.json',
```

Comment:
I think this should be hosted either in a src test folder of this repository or its own repository in KaotoIO.
Also usually a good idea to point to a tag so that the test does not break on main branch here because of changes on the other repository

Reply:
By default, we're not setting a catalog URL, so what about testing adding `http://localhost:5173/camel-catalog/index.json` instead? The advantage would be that while we are still checking that we can set a URL, we benefit from using what's already provided.

Reply:
Update: I tested it and it works :+1:

Reply:
yes, this should be also usable - changed the test to use `http://localhost:5173/camel-catalog/index.json` by default